db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/task/helper/RecurringTask.java

```diff
@@ -57,4 +57,9 @@ public void onStartup(Scheduler scheduler, Clock clock) {
 
     public abstract void executeRecurringly(TaskInstance<T> taskInstance, ExecutionContext executionContext);
 
+    @Override
+    public String toString() {
+        return "RecurringTask name=" + getName() + ", scheduled using " + onComplete;
```

Comment:
Alternatively
```
"RecurringTask name=" + getName() + " with onComplete=" + onComplete;
```

Reply:
Yes, I agree. That may be, strictly speaking, more correct, as a task itself does not know anything about it actually being scheduled or not. Maybe even just
```suggestion
        return "RecurringTask name=" + getName() + ", onComplete=" + onComplete;
```
to align with the format of other toStrings, I don't know if you have any preference on that. I myself tend to like some small bits of "prose"/sentence-like toStrings, instead of enumerating comma-separated key/value pairs, as I think it make them easier to read in log messages, but that is an individual preference of mine.